docs/supported-libraries.md

```diff
@@ -225,6 +225,15 @@ For this reason, the following instrumentations are disabled by default:
 - `jdbc-datasource` which creates spans whenever the `java.sql.DataSource#getConnection` method is called.
 - `dropwizard-metrics` which might create a very low quality metrics data, because of lack of label/attribute support
   in the Dropwizard metrics API.
+- `jaxws-jws-api`
```

Comment:
`jaxws-jws-api`, `mybatis` and `jaxrs` are disabled because they create only internal spans that don't have any defined semantic conventions